app/services/metrics/taxonomy_metrics.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../metrics'
+
+module Metrics
+  class TaxonomyMetrics
```

Comment:
All the metrics in this namespace are currently about the taxonomy, so maybe this class could be called something more specific, TaxonsPerLevelMetrics perhaps?

Reply:
fixed